src/databricks/labs/ucx/install.py

```diff
@@ -622,9 +654,20 @@ def _upload_wheel(self) -> str:
             remote_wheel = f"{self._install_folder}/wheels/{local_wheel.name}"
             remote_dirname = os.path.dirname(remote_wheel)
             with local_wheel.open("rb") as f:
-                self._ws.dbfs.mkdirs(remote_dirname)
-                logger.info(f"Uploading wheel to dbfs:{remote_wheel}")
-                self._ws.dbfs.upload(remote_wheel, f, overwrite=True)
+                try:
+                    self._ws.dbfs.mkdirs(remote_dirname)
+                    logger.info(f"Uploading wheel to dbfs:{remote_wheel}")
+                    self._ws.dbfs.upload(remote_wheel, f, overwrite=True)
+                except OperationFailed as err:
```

Comment:
this is an incorrect exception to catch - it's not thrown in the `dbfs.upload()` code path. 

you have to catch `databricks.sdk.errors.mapping.PermissionDenied` (i think) - we don't have an environment to reproduce this.

Reply:
Just pushed this.